dist/assets/vendor.css

```diff
@@ -1572,13 +1645,13 @@ body {
   line-height: 1em;
   white-space: normal;
   transform: rotateZ(0);
-  min-width: 22em;
+  min-width: 15em;
   height: 3em;
   background-color: #fff;
-  color: #000;
+  color: #474c4f;
```

Comment:
This colour gets called out a couple of places. Should it get a variable/name?

Reply:
oh...the `vendor.css` file is a computed file that converts the variable names i have in the `tailwind/components` directory into the actual css value.

Reply:
AH! OK!